tensorrt_llm/_mnnvl_utils.py

```
@@ -180,6 +180,121 @@ def new_mnnvl_memory_address(cls, mapping: Mapping, size: int):
+            # ---- fast path: pidfd_getfd available ----
+            all_pids = comm.allgather(os.getpid())
+            for pid in all_pids:
+                pidfd = syscall(SYS_pidfd_open, pid, 0)
+                if pidfd < 0:
+                    err = ctypes.get_errno()
+                    raise RuntimeError(
+                        f"pidfd_open({pid}) failed with errno {err}: {os.strerror(err)}"
+                    )
+                pidfds.append(pidfd)
+
+            for pidfd, fd in zip(pidfds, all_handles_data):
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)